nlmod/dims/grid.py

```diff
@@ -1536,7 +1542,7 @@ def gdf_to_count_da(gdf, ds, ix=None, buffer=0.0, **kwargs):
 
     # build list of gridcells
     if ix is None:
-        modelgrid = modelgrid_from_ds(ds)
+        modelgrid = modelgrid_from_ds(ds, rotated=False)
```

Comment:
Just checking if I understand correctly: You convert the gdf to modelcoordinates if you have a rotated grid. Then you use `rotated=False` for the modelgrid because you need a modelgrid with modelcoordinates (and not real world coördinates).

Reply:
Yes